spec/api.yaml

```diff
@@ -0,0 +1,269 @@
+openapi: "3.0.2"
+info:
+  title: Data Connect API
+  version: 1.0.0
+  description: |
+    Data Connect is a standard for discovery and search of biomedical data.
+
+    More information on [GitHub](https://github.com/ga4gh-discovery/data-connect).
+  license:
+    name: 'Apache 2.0'
+    url: 'https://raw.githubusercontent.com/ga4gh-discovery/data-connect/develop/LICENSE'
+  contact:
+    name: 'Data Connect Team'
+    email: 'ga4gh-discovery-search@ga4gh.org'
+security:
+  - bearerAuth: [ ]
+paths:
+  /tables:
+    get:
+      summary: List Tables
+      description: Returns a list of Tables.
+      operationId: listTables
+      tags:
+        - Table API
+      responses:
+        '200':
+          description: A list of Tables
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ListTablesResponse"
+        '500':
+          description: An unexpected error occurred
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
+  /table/{table_name}/info:
+    get:
+      summary: Get a Table
+      description: Returns the information about the Table identified by name.
+      operationId: getTable
+      tags:
+        - Table API
+      parameters:
+        - name: table_name
+          in: path
+          description: Unique Table name, unique to this implementation
```

Comment:
Minor: this wording is a bit awkward, and it's repeated several times in this doc.

Maybe better to say "A table name"?

We could mention the unique thing on the `Table.name` property. We could also provide examples there such as `my_schema.my_table` and `my_catalog.my_schema.my_table` to make it clear that these are unique, qualified names.